lib/entice/logic/movement.ex

```diff
@@ -8,7 +8,7 @@ defmodule Entice.Logic.Movement do
   Note that velocity is actually a coefficient for the real velocity thats used inside
   the client, but for simplicities sake we used velocity as a name.
   """
-  defstruct goal: %Coord{}, plane: 1, move_type: 9, velocity: 1.0
+  defstruct goal: %Coord{}, plane: 1, move_type: 9, velocity: 1.0, update_self: false, update_delay: 1
```

Comment:
I was thinking maybe we can shorten this a bit - for starters, I guess the update interval is static and can be compile with `@update_interval 1` (also note the naming... delay sounds like waiting for something, which we don't do)
Second, I think 1ms for the interval is too short, considering that all NPCs are doing it and there will spam the whole map with position updates. I think for clients we have something like a position update every 50ms or so.
Next, maybe we can find a better name for `update_self`? Something that tells us that this behaviour is automatically recalculating its state?